docker/docker.go

```diff
@@ -124,9 +125,7 @@ func ExecLogin(serverAddress, username, token string) error {
 		return errors.Errorf("Error saving credentials: %v", err)
 	}
 
-	if response.Status != "" {
-		return errors.Errorf("Error saving credentials: %v", response.Status)
-	}
+	fmt.Println(responseOk.Status)
```

Comment:
@andriisoldatenko what does this print now?

Reply:
```
Login Succeeded
```

full output:
```
 Switched cluster
Username (leave blank for oAuth):

Please visit the following URL, authenticate and paste token in next prompt

https://app.staging.astronomer.io/token

oAuth Token: your-token
Default "w1" (ck7bzdcfm09020915a95tjmc0) workspace found, setting default workspace.
Login Succeeded
Successfully authenticated to registry.staging.astronomer.io
```

Reply:
similar to:
```
docker login
Authenticating with existing credentials...
Login Succeeded
```

Reply:
What does it print with username / password instead of OAuth?

Reply:
Were your docker login credentials cleared / cleaned prior to testing?

Reply:
Were any subsequent commands run to test auth actually succeeded? (rather than just printing it did)

I had issues with `astro deployment create default -e local`